oak_attestation_verification/src/verifier.rs

```diff
@@ -34,11 +40,12 @@ const ADDITIONAL_DATA: &[u8] = b"";
 
 /// Verifies entire setup by forwarding to individual setup types.
 pub fn verify(
+    now_utc_millis: i64,
```

Comment:
IMO think this is correct, we should expect the caller to provide the correct time (e.g. it may be that several checks are performed, and we definitely don't want each method to get its own time, to avoid TOCTOU issues). Though I would prefer if it was indeed type safe(r) (i.e. some better type than i64).

Reply:
I agree. It is better to keep the lib now()-free, it will be more robust and also simpler.